packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -321,9 +321,18 @@ export function objectSchema<
 
 	class CustomObjectNode extends CustomObjectNodeBase<T> {
 		public static readonly fields: ReadonlyMap<string, FieldSchema> = new Map(
-			[...flexKeyMap].map(([key, value]) => [key as string, value.schema]),
+			Array.from(flexKeyMap, ([key, value]) => [key as string, value.schema]),
 		);
 		public static readonly flexKeyMap: SimpleKeyMap = flexKeyMap;
+		public static readonly storedKeyToPropertyKey: ReadonlyMap<FieldKey, string> = new Map<
```

Comment:
I believe we already cache such a mapping somewhere. We should dedupe. @alexvy86 @noencke do either of you remember where that existing mapping lives?

Reply:
We have the inverse of this (lthe line above it) but nothing in this direction that I'm aware of.

There is another open PR that also adds it ( https://github.com/microsoft/FluidFramework/pull/22043/files#diff-978f95b1fbda78856fe9c4c79e72a61dce91cf7bbdd8b3fcf0e6c9dbf6685165R55 ), but I don't think it exists in main.

Reply:
Ah, that PR is what I was thinking of. Okay. I guess we should just make sure we don't accidentally duplicate solutions between the PRs.